pom.xml

```diff
@@ -26,6 +26,14 @@
     <description>Tock, The Open Conversation Kit</description>
     <url>https://theopenconversationkit.github.io/tock</url>
 
+
+    <repositories>
```

Comment:
à revert ?

Reply:
hmmm. We should use the local git repository version jar https://github.com/theopenconversationkit/tock/tree/master/nlp/entity-evaluator/duckling/service/lib

Please remove this repo :)